process_options.rb

```diff
@@ -0,0 +1,20 @@
+require './app'
+
+def process_options(usr_inp)
+  case usr_inp
+  when 1
+    @app.list_all_books
+  when 2
+    @app.list_all_people
+  when 3
+    @app.create_person
+  when 4
+    @app.create_book
+  when 5
+    @app.create_rental
+  when 6
+    @app.list_rentals
+  else
+    puts "\e[31mInvalid selectin! please try again."
+  end
```

Comment:
- [ ] @Team I am unable to access any option in your menu, below is the error am getting:
![image](https://github.com/lucy-sees/Libra/assets/61408860/2a0b9602-1356-4548-961f-51313730e8ab)
![image](https://github.com/lucy-sees/Libra/assets/61408860/d87b3a87-6308-41f1-ae17-e6621423badd)